client/src/pages/ProfileSetupPage/ProfileSteps/ProfileStep1/ProfileStep1.js

```diff
@@ -0,0 +1,79 @@
+// importing modules
+import React from 'react';
+import { Box, MenuItem, Select, Typography } from '@material-ui/core';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import CancelIcon from '@material-ui/icons/Cancel';
+import { green, red } from '@material-ui/core/colors';
+// importing components
+import ProfileStepHeader from '../../ProfileStepsHeader/ProfileStepsHeader';
+import ProfileStepFooter from '../../ProfileStepsFooter/ProfileStepsFooter';
+
+const ProfileStep1 = ({ exists, step, 
+    username, timezoneName, 
+    timezonesArr, handleNextStep, 
+    handleUserInput, handleUsernameCheck }) => {
+    return(
+    <Box boxShadow={3} className="step1">
+        <ProfileStepHeader
+            step={step}
+            text="Welcome to CalendApp!" 
+        />
+        <Box className="step1__form--wrap">
+            <Box className="step1__form">
+                <Box className="step1__form--input--wrap">
+                    <Typography variant="body1" className="step1__form--label">Create your CalendApp URL:</Typography>
+                    <Box className="step1__form--input--container">
+                        <p className="step1__form--input--placeholder">calendapp.com/</p>
+                        <input 
+                            onChange={handleUsernameCheck}
+                            className="step1__form--input" 
+                            name="username"
+                            autoComplete="off"
+                            value={username}
+                            type="text"
+                        />
+                        { username.length >= 6 && username.length <= 24 && exists === false ? <CheckCircleIcon style={{ color: green[500] }} className="step1__form--input--icon" /> : null }
+                        { username.length >= 6 && username.length <= 24 && exists === true ? <CancelIcon style={{ color: red[800] }} className="step1__form--input--icon" /> : null }
```

Comment:
Should there be a minimum username size and is 24 too short for a maximum?

Reply:
its common for it to be 6 to 30, I can make it as long as 30 or more?